daniddelrio/akads-ph
# Feature/when2meet

How to test:
1) using a tutee account make a request
2) Make sure to have at least 1 checked box (for now) but you can check any naman it doesn't validate yet if the checked boxes amount to the requested time period
3) click the make request, check console if the checked time slots come out True and the not checked ones come out false (they're not labeled but theyre in the right sequence so you can cross check that way

NOTE:
Sa model.py, and the rest of the code referring to the schedule fields of the sessions table, it doesn't coincide wiht the time it's supposed to represent (like the name). The error starts at seventeenToHalf.
Rest assured that's just a naming error and won't really affect the code and functionality. this is just important to remember

here's the variable names in text para madali i copy paste if ever

            sevenToHalf,
            halfToEight,
            eightToHalf,
            halfToNine,
            nineToHalf,  
            halfToTen,      
            tenToHalf,      
            halfToEleven,   
            elevenToHalf,   
            halfToTwelve,   
            twelveToHalf,   
            halfToThirteen, 
            thirteenToHalf, 
            halfToFourteen, 
            fourteenToHalf, 
            halfToFifteen,  
            fifteenToHalf,  
            halfToSixteen,  
            seventeenToHalf,
            halfToEighteen, 
            eighteenToHalf, 
            halfToNineteen, 
            nineteenToHalf, 
            halfToTwenty,
